packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/documents_batch/data_triggers/triggers/dashpay/v0/mod.rs

```diff
@@ -201,7 +187,7 @@ mod test {
         let result = create_contact_request_data_trigger(
             &DocumentCreateTransitionAction::from(document_create_transition).into(),
             &data_trigger_context,
-            None,
+            PlatformVersion::first(),
```

Comment:
It's best to have this at the top of the test so it's visible and obvious.

Reply:
I will fix it